bin/install_media_pack.ps1

```diff
@@ -0,0 +1 @@
+Install-WindowsFeature Server-Media-Foundation
```

Comment:
Does it work on regular win 10 (non server)?

Reply:
Should work! At least according to stack overflow